### DISCUSSION TITLE
Candlestick charts

### DISCUSSION THREAD

**Discussion by Physicworld** [alandefreitas/matplotplusplus#202] [Ideas]
would be nice if you add candlestick charts to your library.
This is the matplotlib for c++ im open to collaborate im a c++ coder.

Congrats guys! excellent job!

**Comment by alandefreitas** [comment]
Thanks! Candlestick charts would be nice indeed. They are quite difficult to implement though. 

Maybe as a combination of bar plots and error plots, but I'm not sure.
